keyboards/tada68/keymaps/onelivesleft/config.h

```diff
@@ -0,0 +1,4 @@
+#include "../../config.h"
```

Comment:
The keyboard-level `config.h` file is automatically included by QMK.

This line should be changed to:

```suggestion
#pragma once
```